usr/lib/systemd/system/greenboot.target

```diff
@@ -0,0 +1,18 @@
+#  This file is part of greenboot.
+#
+#  greenboot is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=greenboot Success Target
+Wants=greenboot.service
```

Comment:
@jlebon re: https://github.com/LorbusChris/greenboot/pull/4#discussion_r195443658
change this to `Requires` to also fail the target on green script failures, not just health check failures, creating a circular dependency between both the service and the target.